src/components/IngredientListItem.jsx

```diff
@@ -0,0 +1,83 @@
+// Custom App Bar with a "Recipes" title and clickable SearchIcon,
+// which is responsible for showing Search Bar. The component has
+// been implemented on the basis of: https://material-ui.com/components/app-bar/
+
+
+
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import styled from 'styled-components'
+import Checkbox from '@material-ui/core/Checkbox';
+
+
+const PurpleCheckBox = styled(Checkbox)`
+            color: #6200EE !important;
+            backgroundColor: #6200EE !important;
+`
+
+
+const shortenString = (value, maxlen) => {
+  return value.length > maxlen
+    ? value.slice(0, maxlen-2) + "..."
+    : value;
+}
```

Comment:
In my opinion it would be better if you wrap the text instead of shortening it.

Reply:
Add ticket on Jira for that one, mainly because this will require some redesign work.